avm/res/network/trafficmanagerprofile/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -14,6 +14,12 @@ param resourceGroupName string = 'dep-${namePrefix}-network.trafficmanagerprofil
 @description('Optional. The location to deploy resources to.')
 param resourceLocation string = deployment().location
```

Comment:
Can we use the primary enforced location for all dependencies as well? So that we can remove this parameter

Reply:
Removed the resourceLocation param and used the primary enforced location